maat-court-data-api/src/main/java/gov/uk/courtdata/dces/service/DebtCollectionService.java

```diff
@@ -35,47 +34,41 @@ public List<String> getFdcFiles(final LocalDate fromDate, final LocalDate toDate
     }
 
     String convertToCorrectDateFormat(LocalDate localDate) {
-
         DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
         log.info("Printing before date " + localDate.toString());
         log.info("Printing after date " + localDate.format(dateTimeFormatter));
         return localDate.format(dateTimeFormatter);
     }
 
-    public boolean saveError(ContributionFileErrorsEntity entity){
+    public void saveError(ContributionFileErrorsEntity entity) {
         contributionFileErrorsRepository.save(entity);
-        return true;
     }
 
-
-    public boolean updateContributionFileReceivedCount(Integer fileId){
-        if(Objects.isNull(fileId)){
+    public boolean updateContributionFileReceivedCount(Integer fileId) {
+        if (Objects.isNull(fileId)) {
             log.info("No associated file was found for contribution");
             return false;
         }
         ContributionFilesEntity filesEntity = getContributionFile(fileId);
-        if ( Objects.nonNull(filesEntity)){
+        if (Objects.nonNull(filesEntity)) {
             filesEntity.incrementReceivedCount();
             filesEntity.setDateReceived(LocalDate.now());
             debtCollectionRepository.updateContributionFilesEntity(filesEntity);
             log.info("Update of file id : {} successful", fileId);
             return true;
-        }
-        else {
+        } else {
             throw new MAATCourtDataException("No file was found for the fdc.");
```

Comment:
I would include the `fileId` in the exception message also.

Reply:
done